--- /opt/victronenergy/dbus-systemcalc-py/delegates/multi.py.orig
+++ /opt/victronenergy/dbus-systemcalc-py/delegates/multi.py
@@ -10,7 +10,8 @@
 
 	@property
 	def connected(self):
-		return self.monitor.get_value(self.service, '/Connected') == 1
+		return (self.monitor.get_value(self.service, '/Connected') == 1) and \
+			(self.monitor.get_value(self.service, '/Mode') != 4) # ignore turned-off multi's, too
 
 	@property
 	def active_input(self):
