*** /opt_orig/victronenergy/gui/qml/PageMain.qml	2025-11-01 15:46:20.355326065 +0100
--- /opt/victronenergy/gui/qml/PageMain.qml	2025-11-01 15:45:30.286964708 +0100
***************
*** 149,159 ****
--- 149,165 ----
  		PageGenset {}
  	}
  
+ 	Component {
+ 		id: ibrservicesPage
+ 		PageIbrServices {}
+ 	}
+ 
  	function addService(service)
  	{
  		var name = service.name
  
  		var page
+ 		console.log("processing service " + name)
  		switch(service.type)
  		{
  		case DBusService.DBUS_SERVICE_MULTI:
***************
*** 162,167 ****
--- 168,175 ----
  		case DBusService.DBUS_SERVICE_MULTI_RS:
  			return;
  		case DBusService.DBUS_SERVICE_BATTERY:
+ 		    if (name == "com.victronenergy.battery.ibrbms")
+ 		        deviceList.append(service, ibrservicesPage.createObject(root, {service: service, bindPrefix: service.name}))
  			page = batteryPage
  			break;
  		case DBusService.DBUS_SERVICE_SOLAR_CHARGER:
