--- /opt/victronenergy/gui/qml/PageMain.qml.orig	2025-11-01 16:38:36.668213844 +0100
+++ /opt/victronenergy/gui/qml/PageMain.qml	2025-11-01 16:38:45.584284924 +0100
@@ -149,11 +149,17 @@
 		PageGenset {}
 	}
 
+	Component {
+		id: ibrservicesPage
+		PageIbrServices {}
+	}
+
 	function addService(service)
 	{
 		var name = service.name
 
 		var page
+		console.log("processing service " + name)
 		switch(service.type)
 		{
 		case DBusService.DBUS_SERVICE_MULTI:
@@ -162,6 +168,8 @@
 		case DBusService.DBUS_SERVICE_MULTI_RS:
 			return;
 		case DBusService.DBUS_SERVICE_BATTERY:
+		    if (name == "com.victronenergy.battery.ibrbms")
+		        deviceList.append(service, ibrservicesPage.createObject(root, {service: service, bindPrefix: service.name}))
 			page = batteryPage
 			break;
 		case DBusService.DBUS_SERVICE_SOLAR_CHARGER:
